packages/vue/src/components/FPaginator/FPaginator.cy.ts

```diff
@@ -0,0 +1,42 @@
+import { type DefineComponent, defineComponent } from "vue";
+import { FPaginateDataset } from "../FPaginateDataset";
+import { FPaginator } from ".";
+
+function createComponent(): DefineComponent {
+    return defineComponent({
+        template: /* HTML */ `
+            <f-paginate-dataset :items="rows" :itemsPerPage="1">
+                <template #default="{ items: currentPageItems }">
+                    <f-paginator :currentPage="7" />
+                </template>
+            </f-paginate-dataset>
+        `,
+        data() {
+            return {
+                rows: Array.from({ length: 15 }),
+            };
+        },
+        components: {
+            FPaginateDataset,
+            FPaginator,
+        },
+    });
+}
+
+const widths = [
+    { width: 400, type: "page counter" },
+    { width: 800, type: "page buttons" },
+];
+
+widths.forEach(({ width, type }) => {
```

Comment:
```suggestion
for (const { width, type } of widths) {
```

Motivering: `Array.forEach()` är konceptuellt flawed, den måste jobba med side-effects för att fungera och den är ofta mer svårläst och inte lika vedertagen som en vanlig enkel for-slinga.

Men sen vet jag inte om det nog nästan är lika enkelt att bara lägga in describe/it manuellt här? Det är så få varianter (två) att jag nog mest tycker det är mer komplext med loopar, string-interpolation i describe, osv.

```ts
describe("visual regression", () => {
    it("should have approved design for desktop", () => {
        cy.viewport(800, 80);
        cy.mount(FPaginator, { ... });
        cy.toMatchScreenshot();
    });

    it("should have approved design for mobile", () => {
        cy.viewport(400, 80);
        cy.mount(FPaginator, { ... });
        cy.toMatchScreenshot();
    });
});
```

Det blir kort och koncist, tror alla kan läsa och förstå utan större problem (oavsett disciplin eller erfarenhet).